### DISCUSSION TITLE
Vs code snippets for twig in wintercms

### DISCUSSION THREAD

**Discussion by Anshul-code** [wintercms/winter#929] [Ideas]
I am a big fan of Winter CMS and October CMS. Using winter cms now a days as it is open source.
When I am working on a october cms project and editing files in vs code with twig extension enabled, I am able to use snippets.

like I type for and ctrl + space it shows all snippets for that keyword in twig.

But when working in winter cms and wintercms extension enabled. 
I am not able to use twig syntax in .htm files. 

Is there a way to get twig snippets for winter cms template files ?

**Comment by LukeTowers** [comment]
@bennothommo this sounds interesting
